diff --git a/Interface/AddOns/alaChat_Classic/addon.lua b/Interface/AddOns/alaChat_Classic/addon.lua
index 2666ec65..7566ab0d 100644
--- a/Interface/AddOns/alaChat_Classic/addon.lua
+++ b/Interface/AddOns/alaChat_Classic/addon.lua
@@ -163,8 +163,8 @@ local default = {
 	_version				 = 200226.0,
 	_overrideVersion		 = 200226.0,
 
-	printWel				 = true,
-	position				 = "BELOW_EDITBOX",
+	printWel				 = false,
+	position				 = "ABOVE_EDITOBX",
 	direction				 = "HORIZONTAL",
 	scale					 = 0.9,
 	alpha					 = 1.0,
@@ -176,9 +176,9 @@ local default = {
 	chatEmote				 = true,
 	chatEmote_channel		 = true,
 	ColorNameByClass		 = true,
-	shamanColor				 = false,
+	shamanColor				 = true,
 	-- SAY, PARTY, RAID, RAID_WARNING, INSTANCE_CHAT, GUILD, YELL, WHISPER, OFFICER, GENERAL, TRADE, LOCAL_DEFENSE, LOOK_FOR_GROUP, "WORLD"
-	channelBarChannel		 = { true,true,true,true,true,true,true,false,false,true,true,false,true,true },
+	channelBarChannel		 = { true,true,true,false,false,true,true,false,false,false,false,false,false,false },
 	channelBarStyle			 = "CHAR",
 
 	channel_Ignore_Switch 	 = true,
@@ -195,14 +195,14 @@ local default = {
 	DBMCountDown			 = true,
 	ReadyCheck				 = true,
 	statReport				 = true,
-	level					 = false,
+	level					 = true,
 	copy					 = true,
 	copyTagColor			 = { 0.25, 0.25, 1.00 },
 	copyTagFormat			 = "[%H:%M:%S]";
 
 	--hideConfBtn				 = true,
 	editBoxTab				 = true,
-	restoreAfterWhisper		 = false,
+	restoreAfterWhisper		 = true,
 	restoreAfterChannel		 = false,
 	hyperLinkHoverShow		 = true,
 
