diff --git a/Interface/AddOns/alaChat_Classic/addon.lua b/Interface/AddOns/alaChat_Classic/addon.lua
index b2fad5e..1b5763c 100644
--- a/Interface/AddOns/alaChat_Classic/addon.lua
+++ b/Interface/AddOns/alaChat_Classic/addon.lua
@@ -144,8 +144,8 @@ local default = {
 	_version				 = 190830.0,
 	_overrideVersion		 = 200212.0,
 
-	printWel				 = true,
-	position				 = "BELOW_EDITBOX",
+	printWel				 = false,
+	position				 = "ABOVE_EDITOBX",
 	direction				 = "HORIZONTAL",
 	scale					 = 1.0,
 	alpha					 = 1.0,
@@ -157,8 +157,8 @@ local default = {
 	chatEmote				 = true,
 	chatEmote_channel		 = true,
 	ColorNameByClass		 = true,
-	shamanColor				 = false,
-	channelBarChannel		 = { true,true,true,true,true,true,true,true,true,true,true,true,true,true },
+	shamanColor				 = true,
+	channelBarChannel		 = { true,true,true,false,false,true,true,false,false,false,false,false,false,false },
 	channelBarStyle			 = "CHAR",
 
 	channel_Ignore_Switch 	 = true,
@@ -175,14 +175,14 @@ local default = {
 	DBMCountDown			 = true,
 	ReadyCheck				 = true,
 	statReport				 = true,
-	level					 = false,
+	level					 = true,
 	copy					 = true,
 	copyTagColor			 = { 0.25, 0.25, 1.00 },
 	copyTagFormat			 = "[%H:%M:%S]";
 
 	--hideConfBtn				 = true,
 	editBoxTab				 = true,
-	restoreAfterWhisper		 = false,
+	restoreAfterWhisper		 = true,
 	restoreAfterChannel		 = false,
 	hyperLinkHoverShow		 = true,
 
