diff --git a/Interface/AddOns/ClassicAuraDurations/code.lua b/Interface/AddOns/ClassicAuraDurations/code.lua
index fe04dd9..9087d30 100644
--- a/Interface/AddOns/ClassicAuraDurations/code.lua
+++ b/Interface/AddOns/ClassicAuraDurations/code.lua
@@ -41,10 +41,10 @@ end
 local defaults = {
     portraitIcon = true,
     playerPortraitIcon = true,
-    enemyBuffs = false,
+    enemyBuffs = true,
     hookTargetFrame = true,
     verbosePortraitIcon = false,
-    largePersonalDebuffs = false,
+    largePersonalDebuffs = true,
 }
 
 -- Redefining blizzard consts
