diff --git a/Interface/AddOns/ClassicCastbars/core/Data.lua b/Interface/AddOns/ClassicCastbars/core/Data.lua
index 78f08893..b9a44600 100644
--- a/Interface/AddOns/ClassicCastbars/core/Data.lua
+++ b/Interface/AddOns/ClassicCastbars/core/Data.lua
@@ -1798,7 +1798,7 @@ namespace.defaultConfig = {
     usePerCharacterSettings = false,
 
     nameplate = {
-        enabled = true,
+        enabled = false,
         showForFriendly = true,
         showForEnemy = true,
         width = 106,
